additional-databases/neptune/README.md

```diff
@@ -0,0 +1,38 @@
+## Connecting a local graph-notebook to Amazon Neptune (first-time setup)
+When using graph-notebook locally to connect to an Amazon Neptune database for the first time, there are a couple of additional steps. This section assumes that you've already installed & configured [graph-notebook](#installation) locally.  Please note that this wiki is not an official recommendation on network setups as there are many ways to connect to Amazon Neptune from outside of the VPC, such as setting up a load balancer or VPC peering.
```

Comment:
the hyper-link \[graph-notebook](#installation) likely will not work now since we have moved this to another directory. It should now be \[graph-notebook](https://github.com/aws/graph-notebook#installation)